engine/table/src/main/java/io/deephaven/engine/table/impl/select/SelectColumn.java

```diff
@@ -68,7 +68,9 @@ static Collection<SelectColumn> copyFrom(Collection<SelectColumn> selectColumns)
      * @param table the table to initialize internals from
      * @return a list containing all columns from 'table' that the result depends on
      */
-    List<String> initInputs(Table table);
+    default List<String> initInputs(Table table) {
```

Comment:
`@FinalDefault`?

Reply:
Or mabye just delete it?

Reply:
Deleting it and inlining the uses would be OK, I guess.